src/utils/options.js

```diff
@@ -31,7 +31,9 @@ const baseSchema = {
                     "type": "object",
                     "oneOf":[ 
                         { "required": ["field", "type", "contains"] },
-                        { "required": ["field", "type", "range"] }
+                        { "required": ["field", "type", "range"] },
+                        { "required": ["field", "type", "subtree"] },
+                        { "required": ["field", "type", "minSimilarity"] },
```

Comment:
Currently, this is not implemented because our interface do not support applying multiple highlighting options for individual fields in the interface (i.e., new highlight option replace the previous one). But, if we find this use case to be essential, I think we can implement that with a separate PR.